tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -621,6 +623,37 @@ function setupStorybook(tree: Tree, options: NormalizedSchema) {
     });
   }
 
+  function removeTsIgnorePragmas() {
```

Comment:
should this mark this to be removed after migration ? I don't see any point of keeping it there after